pkg/build/path_rewriter.go

```diff
@@ -22,23 +22,20 @@ import (
 // RelativePathRewriter rewrites paths when the paths are relative paths.
 type RelativePathRewriter struct{}
 
-// MakeAbs rewrites file paths if the path is relative, from the
+// AbsWithParent rewrites file paths if the path is relative, from the
 // original object path, to an object path that's based on a parent component
 // path. Note that the parent path must be an absolute path.
 //
 // For example, if the component path is foo/bar/biff.yaml and the object path
 // is zed/fred.yaml, the object will be rewritten as foo/bar/zed/fred.yaml
-func (rw *RelativePathRewriter) MakeAbs(parent, obj *url.URL) *url.URL {
+func (rw *RelativePathRewriter) AbsWithParent(parent, obj *url.URL) *url.URL {
 	if parent == nil || obj == nil {
 		return obj
 	}
 	if parent.Scheme != "file" && parent.Scheme != "" {
 		// Only file schemes are supported.
 		return obj
 	}
-	if !filepath.IsAbs(parent.Path) {
-		return obj
-	}
 	if filepath.IsAbs(obj.Path) {
```

Comment:
Function has behavior of returning the objpath is it's already absolute, disregarding the parent argument.

Reply:
Yes, that seems like the right behavior. If the path is already absolute, then it doesn't need to be made absolute